test/unit/clients/ledger_client/ledger_search_payout_test.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+// NPM dependencies
```

Comment:
I don't think we need these comments in all of our files, this is just a semantic comment but I think it's plenty to separate the (npm) library imports by a new line from the local imports